templates/channel.pug

```diff
@@ -111,9 +111,9 @@ html(lang="en")
                   .input-group
                     input#mediaurl.form-control(type="text", placeholder="Media URL")
                     span.input-group-btn
-                      button#queue_next.btn.btn-default Next
+                      button#queue_next.btn.btn-default Play Next
                     span.input-group-btn
-                      button#queue_end.btn.btn-default At End
+                      button#queue_end.btn.btn-default Add to Playlist
```

Comment:
Looks like there is a second place that needs to be updated

Reply:
Good spot, updated